### PR TITLE
Add Microsoft Quantum brand

### PR DESCRIPTION
**Pull request by cjgronlund**
Add brand to articles
Add language and brand to Q# ref docs

Global settings in docfx file

### CODE REVIEW

File: api/docfx.json [hunk 1]
```diff
@@ -54,6 +54,10 @@
       "feedback_github_repo": "Microsoft/quantum",
       "feedback_product_url": "https://github.com/microsoft/quantum/issues"
     },
+    "titleSuffix": {
+      "*.md": "Q# reference - Microsoft Quantum"
+      }
```
Comment (cgranade) [inline api/docfx.json]: It looks like there is an extra `}` here that is preventing DocFX from provisioning the docset correctly.
```suggestion
```

File: articles/docfx.json [hunk 2]
```diff
@@ -64,6 +64,10 @@
         }
       }
     },
+    "titleSuffix": {
+      "*.md": "Microsoft Quantum"
+      }
```
Comment (cgranade) [inline articles/docfx.json]: Similarly here:
```suggestion
```

File: api/docfx.json (hunk 1 above)
Comment (cjgronlund) [reply api/docfx.json]: Thanks for the heads up. I'll fix and resubmit.

File: articles/docfx.json (hunk 2 above)
Comment (cjgronlund) [reply articles/docfx.json]: Validation complete and fix submitted. Thank you!

**Comment by cgranade** [comment]
Wonderful, thanks for the help, @cjgronlund!

**Pull request merged**